docs/installation/bootstrap.rst

```diff
@@ -206,6 +207,15 @@ defaults kubernetes configuration.
             usernameClaim: <Username Claim>
             groupsClaim: <Groups Claim>
 
+  From ``coreDNS`` section you can disable the ``hostForward`` so that the DNS
+  request from the Pod inside Kubernetes are not forwarded to the configured
+  nameservers from the host.
+
+    .. note::
+
+      It means pod running in Kubernetes will not be able to resolve any name
+      that are not in Kubernetes.
```

Comment:
There still is a way to make it work with `ExternalName` Service objects 😉

Reply:
Of it's in a Service as `ExternalName` then it's in Kubernetes :smiley: